jwst/outlier_detection/outlier_detection_step.py

```diff
@@ -45,11 +44,13 @@ class OutlierDetectionStep(Step):
 
     """
 
+    # FIXME note that this step does not have a suffix, the image3, coron3
```

Comment:
I would say yes it's still relevant.

I just tested this with calwebb_image3 and just outlier_detection. Since the step doesn't have a suffix, the output file has a different suffix when run within a pipeline vs run outside the pipeline. For outlier detection, the different pipelines set different suffixes:
```
calwebb_coron3.py:        self.outlier_detection.suffix = 'crfints'
calwebb_image3.py:        self.outlier_detection.suffix = 'crf'
calwebb_spec3.py:        self.outlier_detection.suffix = 'crf'
```
but run outside the pipeline the suffix will be `outlierdetectionstep` (for all "modes"). As `calwebb_tso3` doesn't set a suffix it will default to `outlierdetectionstep`.
I see no mention of the `outlierdetectionstep` suffix in the docs related to outlier detection results:
https://jwst-pipeline.readthedocs.io/en/latest/jwst/data_products/science_products.html#cosmic-ray-flagged-data-crf-and-crfints
So I would consider this a bug both in tso3 and the step.

I think this PR has already done maybe too much and fixing the above bugs could go into follow-up PRs (the main goal of this PR is achieved without fixing those bugs).

If preferred I can remove this `#FIXME` and open issues to track the bugs.

Reply:
I removed the `#FIXME` in d26cc5a37

I'll open issues to track the bugs.